### PR TITLE
Update NU1605.md

### PR DESCRIPTION
**Pull request by nkolev92**
Per the CLI/NuGet sync-up. 

We need better guidance for customers. 

cc @dsplaisted @nguerrera @NuGet/nuget-client

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[dffbd60](https://github.com/NuGet/docs.microsoft.com-nuget/commits/dffbd60fdd00393337ce59ab9c13a2cf70b6500f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/errors-and-warnings/NU1605.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master-nkolev92-nu/docs/reference/errors-and-warnings/NU1605.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1605?branch=pr-en-us-1706) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C26%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201909260019329089-1706%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=T42iQ9m%2FpaHOfO2APcg8Qk8q1X9ctikHtBgCd0VDdaA%3D&st=2019-09-26T00%3A15%3A33Z&se=2019-10-27T00%3A20%3A33Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by dsplaisted** [comment]
As suggested by @nkolev92, we should add `PrivateAssets="All"` to the suggested PackageReference to add.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[b9c21fa](https://github.com/NuGet/docs.microsoft.com-nuget/commits/b9c21fa77cb26855c5dd8dbaeab1864c1e38012f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/errors-and-warnings/NU1605.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master-nkolev92-nu/docs/reference/errors-and-warnings/NU1605.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1605?branch=pr-en-us-1706) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C26%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201909261916333669-1706%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=C3o94kk48SNRLIg4YMY893mLLMAnFlRd29%2FZnZ3kDac%3D&st=2019-09-26T19%3A12%3A29Z&se=2019-10-27T19%3A17%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/errors-and-warnings/NU1605.md [hunk 1]
```diff
@@ -44,3 +44,22 @@ The mvc project specified a version constraint on a higher version of a package
...
+## Example 3
+<pre>
+Detected package downgrade: System.IO.FileSystem.Primitives from 4.3.0 to 4.0.1. Reference the package directly from the project to select a different version.</br>
+ Project -> System.IO.FileSystem 4.0.1 -> runtime.win.System.IO.FileSystem 4.3.0 -> System.IO.FileSystem.Primitives (>= 4.3.0)</br>
+ Project -> System.IO.FileSystem 4.0.1 -> System.IO.FileSystem.Primitives (>= 4.0.1)</br>
+</pre>
+
+### Issue 
+
+Certain combinations of packages which shipped with .NET Core 1.0 and 1.1 are not compatible with each other when they are referenced together in a .NET Core 3.0 or higher project, and a RuntimeIdentifier is specified.  The problematic packages generally start with System or Microsoft, and have version numbers between 4.0.0 and 4.3.1.  In this case, the downgrade message will have a package starting with runtime.<RID> in the dependency chain.
```
Comment (zivkan) [inline docs/reference/errors-and-warnings/NU1605.md]:
```suggestion
Certain combinations of packages which shipped with .NET Core 1.0 and 1.1 are not compatible with each other when they are referenced together in a .NET Core 3.0 or higher project, and a RuntimeIdentifier is specified.  The problematic packages generally start with `System.` or `Microsoft.`, and have version numbers between 4.0.0 and 4.3.1.  In this case, the downgrade message will have a package starting with runtime.<RID> in the dependency chain.
```

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[5c0794a](https://github.com/NuGet/docs.microsoft.com-nuget/commits/5c0794afa7be935ef1dc1807e8f7e2193f0cf5a8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/errors-and-warnings/NU1605.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master-nkolev92-nu/docs/reference/errors-and-warnings/NU1605.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1605?branch=pr-en-us-1706) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C26%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201909262120455364-1706%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=LmBs9vPJmr5AVjA4AzWKj5%2BrajIkROUI4EuMsH4QG9g%3D&st=2019-09-26T21%3A16%3A42Z&se=2019-10-27T21%3A21%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by dsplaisted** [comment]
@rrelyea yes and yes would be my opinion.

**Comment by nkolev92** [comment]
I like that idea as well. 

I switched the examples' order.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[6a34dbe](https://github.com/NuGet/docs.microsoft.com-nuget/commits/6a34dbea8930ed4866e5d0d87d35c6e60161322a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/errors-and-warnings/NU1605.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master-nkolev92-nu/docs/reference/errors-and-warnings/NU1605.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1605?branch=pr-en-us-1706) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C30%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201909301730108069-1706%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=mJPDbKvqoCcFZbPkTlI7%2F75ENK7wVMLx%2F3bUMVqgiIA%3D&st=2019-09-30T17%3A26%3A08Z&se=2019-10-31T17%3A31%3A08Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by nkolev92** [comment]
@NuGet/nuget-client , @karann-msft  can someone take a look at this please :)
It'd be nice to get it merged :)

**Pull request merged**